worker/games.py

```diff
@@ -310,6 +310,14 @@ def parse_cutechess_output(p, remote, result, spsa, spsa_tuning, games_to_play,
       kill_process(p)
       return { 'task_alive': True }
 
+    # Parse line like this:
+    # Warning: New-eb6a21875e doesn't have option ThreatBySafePawn
+    if line.split()[0]=='Warning:':
+      message = r'Cutechess-cli says: "%s"' % line.strip()
+      result['message']=message
+      send_api_post_request(remote + '/api/stop_run', result).json()
+      raise Exception(message)
+
```

Comment:
The signature code also has this fragment:

```
finally:
    if concurrency > 1:
      busy_process.communicate(enc('quit\n'))
      busy_process.stdin.close()
```

Not sure if that is really needed.

Reply:
I don't think so, it is only needed for the signature check.

Reply:
I will think about it... It has to do with cleaning up.